### DISCUSSION TITLE
tea: error: we currently don't support macOS versions less than 11

### DISCUSSION THREAD

**Discussion by ronnakamoto** [pkgxdev/pkgx#260] [Help] [answered]
Was trying to do `sh <(curl https://tea.xyz)` but bumped into this issue:

`tea: error: we currently don't support macOS versions less than 11`

Is there any workaround for this other than upgrading the OS?

I'm currently on macOS Mojave 10.14.6

Thanks!

**Comment by mxcl** [comment, marked as answer]
You can install manually: https://github.com/teaxyz/cli#now-see-here-fella-i-hate-installers

```sh
$ sudo install -m 755 <(curl --compressed -LSsf https://tea.xyz/$(uname)/$(uname -m)) /usr/local/bin/tea
```

Most stuff will work, some things will be broken. It's hard to know *specifically* what will be broken.

Potentially we could set a lower deployment target for our builds (/cc @jhheider)

**Reply by ronnakamoto** [reply]
Thanks, @mxcl. Though that didn't work for me out of the box. 

`install: /dev/fd/63: Bad file descriptor`

The link helped me.

Here is what I did to get started with tea:

- `curl https://dist.tea.xyz/tea.xyz/darwin/x86-64/v0.16.0.tar.gz | tar xzf -`
- `./tea --sync`

**Reply by mxcl** [reply]
Possibly because your shell doesn’t like `<()` syntax. We should rework that one-liner to not need it, but I don’t think there's a way TBH. `install` requires a file.

**Reply by mxcl** [reply]
We chose macOS 11 to reduce the matrix of potential issues, but over time we can expand that. The thing is over time we naturally expand since new macOS versions are released and the number of people on older macOS decreases.
